MeshLib/MeshGenerators/VtkMeshConverter.h

```diff
@@ -80,6 +83,10 @@ class VtkMeshConverter
 	                                    MeshElemType elem_type,
 	                                    UseIntensityAs intensity_type);
 
+	static void convertScalarArrays(vtkUnstructuredGrid &grid, MeshLib::Mesh &mesh);
+
+	static void convertArray(vtkDataArray* array, MeshLib::Properties &properties, MeshLib::MeshItemType type);
```

Comment:
could you please add a little bit of documentation to the tests? (i.e. what is actually tested)
all my other comments are non-critical, you can ignore them if you want to. :)

:+1: (apart from test-docu)